ERROR: TCL interpreter returned an error: Yosys command produced an error

### Subject

[Stage]: Synthesis.

### Describe the bug

I'm trying to follow the "OpenROAD Flow Scripts Tutorial" instructions found at https://openroad-flow-scripts.readthedocs.io/en/latest/tutorials/FlowTutorial.html using the https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildWithDocker.html system.

At the [Verifying the Docker based Installation](https://openroad-flow-scripts.readthedocs.io/en/latest/tutorials/FlowTutorial.html#verifying-the-docker-based-installation) step I'm getting the following output;
```
# ==============================================================================
#  ____  _____ _____ _   _ ____
# / ___|| ____|_   _| | | |  _ \
# \___ \|  _|   | | | | | | |_) |
I have no name!@ebbdfe7a8b0a:/OpenROAD-flow-scripts/flow$ make
/OpenROAD-flow-scripts/flow/scripts/synth.sh /OpenROAD-flow-scripts/flow/scripts/synth_canonicalize.tcl ./logs/nangate45/gcd/base/1_1_yosys_canonicalize.log
1. Executing Verilog-2005 frontend: /OpenROAD-flow-scripts/flow/designs/src/gcd/gcd.v
2. Executing Liberty frontend: ./objects/nangate45/gcd/base/lib/NangateOpenCellLibrary_typical.lib
ERROR: TCL interpreter returned an error: Yosys command produced an error
Command exited with non-zero status 1
Elapsed time: 0:00.15[h:]min:sec. CPU time: user 0.14 sys 0.01 (100%). Peak memory: 32960KB.
make[1]: *** [Makefile:291: do-yosys-canonicalize] Error 1
make: *** [Makefile:294: results/nangate45/gcd/base/1_synth.rtlil] Error 2
I have no name!@ebbdfe7a8b0a:/OpenROAD-flow-scripts/flow$ 
```

### Expected Behavior

Get the following output as described in the document;
```
[INFO] Writing out GDS/OAS 'results/nangate45/gcd/base/6_1_merged.gds'
Elapsed time: 0:10.44[h:]min:sec. CPU time: user 2.17 sys 0.54 (26%). Peak memory: 274184KB.
cp results/nangate45/gcd/base/6_1_merged.gds results/nangate45/gcd/base/6_final.gds
Log                       Elapsed seconds
1_1_yosys                          2
3_3_place_gp                       1
4_1_cts                            8
5_2_route                   10
6_1_merge                         10
6_report                           3
```

### Environment

```shell
The script doesn't seem to work?


I have no name!@6c6a562a4655:/OpenROAD-flow-scripts/tools/OpenROAD$ ./etc/Env.sh 
tee: build/openroad-env.log: Permission denied                                                                                                              
Unknown git commit, this is not a git repository.                                                                                                           
                                                                                                                                                            
Please make sure that you have the latest code changes and add the commit                                                                                   
hash in the description.                                                      
                                                                              
kernel: Linux 6.8.0-55-generic                                                
os: Ubuntu 22.04.4 LTS (Jammy Jellyfish)                      
cmake version 3.24.2                                                          
CMake Warning at CMakeLists.txt:98 (message):                                                                                                               
  OpenROAD git describe failed, using sha1 instead                     
                                                                                                                                                            
                                                                              
CMake Error: Could not open file for write in copy operation /OpenROAD-flow-scripts/tools/OpenROAD/include/ord/Version.hh.tmp                               
CMake Error: : System Error: Inappropriate ioctl for device         
CMake Error at CMakeLists.txt:135 (configure_file):   
  configure_file Problem configuring file                                                                                                                   
                                                                              
                                                                                                                                                            
-- The CXX compiler identification is GNU 11.4.0                                                                                                            
-- Detecting CXX compiler ABI info                                                                                                                          
-- Detecting CXX compiler ABI info - done                       


The `./build_openroad.sh` script produces the following output;

#21 [stage-2  7/11] COPY --link flow/scripts flow/scripts
#21 CACHED

#22 [stage-2  3/11] COPY --link docs docs
#22 CACHED

#23 [stage-2 11/11] COPY --link      --exclude=.git* --exclude=tools/ --exclude=docs/ --exclude=docker/      --exclude=flow/designs --exclude=flow/platforms --exclude=flow/scripts      --exclude=flow/test --exclude=flow/tutorials --exclude=flow/util   . ./
#23 CACHED

#24 exporting to image
#24 exporting layers done
#24 writing image sha256:35fabdf7989da7654b11f10e56ec0cc5fa69843b7afaaacea2e00b7f577211ab done
#24 naming to docker.io/openroad/flow-ubuntu22.04-builder:bc62c0 done
#24 DONE 0.0s
```

### To Reproduce

Follow the instructions @ https://openroad-flow-scripts.readthedocs.io/en/latest/tutorials/FlowTutorial.html#verifying-the-docker-based-installation

Got to the step;
```
docker run --rm -it -u $(id -u ${USER}):$(id -g ${USER}) -v $(pwd)/flow:/OpenROAD-flow-scripts/flow openroad/flow-ubuntu22.04-builder
cd flow
make
```

Wasn't sure what issue command to use, so adding the output of `make synth_issue`

[synth_gcd_nangate45_base_2025-04-03_23-30.tar.gz](https://github.com/user-attachments/files/19596029/synth_gcd_nangate45_base_2025-04-03_23-30.tar.gz)

### Relevant log output

```shell

```

### Screenshots

_No response_

### Additional Context

_No response_

--- a/docs/tutorials/FlowTutorial.md
+++ b/docs/tutorials/FlowTutorial.md
@@ -49,68 +49,24 @@ Use the `bash` shell to run commands and scripts.
 
 #### OpenROAD-flow-scripts Installation
 
-To install OpenROAD-flow-scripts,  refer to the
+To install OpenROAD-flow-scripts, refer to the
 [Build or installing ORFS  Dependencies](https://openroad-flow-scripts.readthedocs.io/en/latest/#build-or-installing-orfs-dependencies)
 documentation.
 
 In general, we recommend using `Docker` for an efficient user
-experience. Install OpenROAD-flow-scripts using a docker as described
-here [Build from sources using Docker](../user/BuildWithDocker.md).
+experience. Please refere to the [Docker Shell](../user/DockerShell.md)
+documentation.
 
 :::{Note}
-If you need to update an existing OpenROAD-flow-scripts installation,
-follow instructions from [here](../user/FAQS.md#how-do-i-update-openroad-flow-scripts).
+If you need a custom Docker image, you can build your own with the instructions
+provided at: [Build from sources using Docker](../user/BuildWithDocker.md).
 :::
 
-OpenROAD-flow-scripts installation is complete.
-
-#### Running OpenROAD-flow-scripts inside the Docker
-
-Launch the docker with OpenROAD-flow-scripts container as follows:
-
-```shell
-docker run --rm -it -u $(id -u ${USER}):$(id -g ${USER}) -v $(pwd)/flow:/OpenROAD-flow-scripts/flow openroad/flow-ubuntu22.04-builder
-```
-
-:::{seealso}
-To launch OpenROAD GUI inside the docker, based on the OS, use the command from [here](../user/BuildWithDocker.md#enable-gui-support).
+:::{Note}
+If you need to update an existing OpenROAD-flow-scripts installation:
+- For local installs follow instructions from [here](../user/FAQS.md#how-do-i-update-openroad-flow-scripts).
+- For Docker run `docker pull openroad/orfs:latest` to update the image.
 :::
-
-Once you are entered into OpenROAD-flow-scripts container run:
-
-```shell
-source env.sh
-```
-
-If your installation is  successful, you will see the following message:
-
-```
-OPENROAD: /OpenROAD-flow-scripts/tools/OpenROAD
-```
-
-#### Verifying the Docker based  Installation
-
-To verify the installation run the built-in example design as follows:
-
-```shell
-cd flow
-make
-```
-
-A successful run end with the log:
-
-```
-[INFO] Writing out GDS/OAS 'results/nangate45/gcd/base/6_1_merged.gds'
-Elapsed time: 0:10.44[h:]min:sec. CPU time: user 2.17 sys 0.54 (26%). Peak memory: 274184KB.
-cp results/nangate45/gcd/base/6_1_merged.gds results/nangate45/gcd/base/6_final.gds
-Log                       Elapsed seconds
-1_1_yosys                          2
-3_3_place_gp                       1
-4_1_cts                            8
-5_2_route                   10
-6_1_merge                         10
-6_report                           3
-```
 
 ## Configuring The Design
 

--- a/docs/user/BuildWithDocker.md
+++ b/docs/user/BuildWithDocker.md
@@ -1,5 +1,11 @@
 # Build from sources using Docker
 
+:::{Note}
+This document describes how to build your own Docker image.
+If your goal is to use the latest version of OpenROAD-Flow-Scripts,
+please refer to the [Docker Shell](./DockerShell.md) documentation.
+:::
+
 ## Prerequisites
 
 - For this method you only need to install
@@ -31,9 +37,9 @@ This greatly helps to reduce the compilation time needed.
 
 We recommend to use a Docker image of a supported OS
 and install OpenROAD using the pre-built binaries from
-Precision Innovations. 
-You can start the container in an interactive mode using 
-the command below. 
+Precision Innovations.
+You can start the container in an interactive mode using
+the command below.
 
 ```shell
 docker run -it ubuntu:22.04
@@ -87,7 +93,7 @@ exit
 ```
 
 Alternatively you may also use the `docker_shell` utility as follows.
-It is important that you are in the `flow` directory. 
+It is important that you are in the `flow` directory.
 
 ```shell
 cd flow
@@ -121,8 +127,8 @@ Then use:
 docker run --rm -it -e DISPLAY=<IP_LIKE_FROM_TUTORIAL>:0 --network host --privileged <IMAGE_NAME>
 ```
 
-Alternatively, you may also use the `docker_shell` utility for GUI as follows. 
-It is important that you are in the `flow` directory. 
+Alternatively, you may also use the `docker_shell` utility for GUI as follows.
+It is important that you are in the `flow` directory.
 
 ```shell
 cd flow
@@ -130,8 +136,8 @@ util/docker_shell gui_final
 ```
 
 ```{note}
-`docker_shell` is a helpful utility to automate the 
-aforementioned Docker commands using the user's parameters. 
+`docker_shell` is a helpful utility to automate the
+aforementioned Docker commands using the user's parameters.
 Do refer to the documentation [here](./DockerShell.md).
 ```
 

--- a/docs/user/DockerShell.md
+++ b/docs/user/DockerShell.md
@@ -1,19 +1,23 @@
 Building example design using Docker image
 ==========================================
 
-The `docker_shell` script is used as a prefix to launch a command using a OpenROAD docker image.
+The `docker_shell` script is used to launch a command using a OpenROAD-flow-scripts docker image.
 
 Also, the current working directory is mapped into the Docker image using the current user's credentials.
 
 Build docker image
 ------------------
 
-First build the docker image:
+You can skip this step if you would like to use the latest version from the master
+branch. If you are developing ORFS/OR, you should build your own image.
 
 ```
 cd OpenROAD-flow-scripts
 ./build_openroad.sh
 ```
+
+Run ORFS using `docker_shell`
+-----------------------------
 
 Build an example design and run the GUI:
 
@@ -33,7 +37,7 @@ If you need to use a different Docker image than default, override by using the 
 environment variable:
 
 ```
-OR_IMAGE=openroad/flow-ubuntu22.04-builder:v1234 util/docker_shell make
+OR_IMAGE=openroad/orfs:v1234 util/docker_shell make
 ```
 
 If you have built your OpenROAD Docker image using prebuilt binaries,

--- a/flow/util/docker_shell
+++ b/flow/util/docker_shell
@@ -62,7 +62,7 @@ docker run $user_args \
  -v $WORKSPACE:/OpenROAD-flow-scripts/flow:Z \
  --network host \
  $DOCKER_INTERACTIVE \
- ${OR_IMAGE:-openroad/flow-ubuntu22.04-builder:latest} \
+ ${OR_IMAGE:-openroad/orfs:latest} \
  bash -c "set -ex
  mkdir /tmp/xdg-run
  cd /OpenROAD-flow-scripts/flow

